docs/01-instalation.md

```diff
@@ -0,0 +1,105 @@
+# Installation
```

Comment:
Can you rename this file to `01-installation.md`? There's an `l` missing

Reply:
Done